sw/airborne/modules/loggers/sdlogger_spi_direct.c

```diff
@@ -109,6 +161,10 @@ void sdlogger_spi_direct_init(void)
  */
 void sdlogger_spi_direct_periodic(void)
```

Comment:
While looking at the frequency of this function in the module file, it is hardcoded to 512 Hz, which is probably not wise. If it is suppose to run at the main frequency, just don't specify the `freq` attribute as it may not be the desired behavior when running at a different frequency.
Otherwise, it looks fine.

Reply:
I agree, but I'm having some trouble getting the logger to work at all today so I'll take another look at this after the christmas break. Happy holidays! ;)

Reply:
I have retested it this morning, everything works fine when freq="512" is present, but after removing it the datalink becomes unstable and the download program has trouble establishing a connection. I am not sure why this happens, but suspect an sd timing issue. I'd rather leave the freq as-is for now, as I don't have time to debug this further.